templates/config.yaml

```diff
@@ -3,4 +3,13 @@ gitRepoId: adyen-go-api-library/v6/src
 structPrefix: true
 isGoSubmodule: true
 additionalProperties:
-  useOneOfDiscriminatorLookup: true
\ No newline at end of file
+  useOneOfDiscriminatorLookup: true
+templateDir: ./templates/custom
```

Comment:
Setting `templateDir` here might take precedence over CLI arguments we pass in the `Makefile`. I would avoid it unless strictly required.

Reply:
I am not sure how to do that. Maybe we could split it in another ticket?